target/linux/bmips/image/bcm63xx.mk

```diff
@@ -0,0 +1,33 @@
+#
+# BCM63xx Profiles
+#
+
+define Device/bcm63xx_cfe
+  FILESYSTEMS := squashfs jffs2-64k jffs2-128k
+  KERNEL := kernel-bin | append-dtb | relocate-kernel | lzma-bcm63xx | lzma-cfe
+  KERNEL_INITRAMFS := kernel-bin | append-dtb | lzma | loader-lzma elf
+  KERNEL_INITRAMFS_SUFFIX := .elf
+  IMAGES := cfe.bin sysupgrade.bin
+  IMAGE/cfe.bin := cfe-bin --pad $$$$(shell expr $$$$(FLASH_MB) / 2)
+  IMAGE/sysupgrade.bin := cfe-bin
+  BLOCK_SIZE := 0x10000
+  IMAGE_OFFSET :=
+  FLASH_MB := 4
+  CHIP_ID :=
+  CFE_BOARD_ID :=
+  CFE_EXTRAS := --block-size $$(BLOCK_SIZE) --image-offset $$(if $$(IMAGE_OFFSET),$$(IMAGE_OFFSET),$$(BLOCK_SIZE))
+endef
+DEVICE_VARS += BLOCK_SIZE FLASH_MB IMAGE_OFFSET
```

Comment:
Most targets use "BLOCKSIZE" which is defined globally in https://github.com/openwrt/openwrt/blob/master/include/image.mk (and added to DEVICE_VARS there).

I wonder whether we can switch to the common name here? And if not, maybe then using a more specific name BLOCKSIZE_CFE or similar would be more appropriate?